plugin/hosts/hostsfile.go

```diff
@@ -117,11 +117,15 @@ func (h *Hostsfile) readHosts() {
 	defer file.Close()
 
 	stat, err := file.Stat()
+	if err != nil {
+		log.Warningf("Failed to get file info: %v", err)
```

Comment:
Perhaps we should skip logging a warning here for the same reason it is skipped above (`// We already log a warning if the file doesn't exist or can't be opened on setup. No need to return the error here.`).

Reply:
You are right. I have removed the warning.